lib/components/viewers/nearby/nearby-filter.tsx

```diff
@@ -0,0 +1,80 @@
+import { Check2 } from '@styled-icons/bootstrap'
+import { Parking } from '@styled-icons/fa-solid'
```

Comment:
```suggestion
import { Check2 } from '@styled-icons/bootstrap/Check2'
import { Parking } from '@styled-icons/fa-solid/Parking'
```
Don't want to break tree shaking!